ui/src/frontend/track_group_panel.ts

```diff
@@ -101,55 +101,77 @@ export class TrackGroupPanel extends Panel<Attrs> {
   get summaryTrackState(): TrackState {
     return assertExists(globals.state.tracks[this.trackGroupState.tracks[0]]);
   }
+
   resize = (e: MouseEvent): void => {
     e.stopPropagation();
     e.preventDefault();
     if (!this.summaryTrack) {
       return;
     }
-    let y = this.summaryTrack.getHeight();
-    const mouseMoveEvent = (evMove: MouseEvent): void => {
-      evMove.preventDefault();
-      y += evMove.movementY;
-      if (this.attrs && this.defaultHeight) {
-        const newMultiplier = y / this.defaultHeight;
-        if (newMultiplier < 1) {
-          this.summaryTrackState.scaleFactor = 1;
-        } else {
-          this.summaryTrackState.scaleFactor = newMultiplier;
-        }
-        globals.rafScheduler.scheduleFullRedraw();
+    if (e.currentTarget instanceof HTMLElement) {
+      const timelineElement = e.currentTarget.closest('div.pan-and-zoom-content');
+      if (timelineElement && timelineElement instanceof HTMLDivElement) {
+        let trackHeight = this.summaryTrack.getHeight();
+        let mouseY = e.clientY;
+        const mouseMoveEvent = (evMove: MouseEvent): void => {
+          evMove.preventDefault();
+          trackHeight += evMove.clientY-mouseY;
+          mouseY = evMove.clientY;
+          if (this.attrs && this.defaultHeight) {
+            const newMultiplier = trackHeight / this.defaultHeight;
+            if (newMultiplier < 1) {
```

Comment:
style nit: 
```
this.summaryTrackState.scaleFactor = Math.min(1, newMultiplier);
```

Five lines of code down to one without, IMO, any loss in readability.